ReactQt/runtime/src/qml/ReactModal.qml

```diff
@@ -0,0 +1,38 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import QtGraphicalEffects 1.0
+import React 0.1 as React
+
+Item {
+
+    id: modalRoot
+    objectName: "modalRoot"
+
+    property string p_animationType
```

Comment:
@vkjr , p_animationType, p_transparent are not implemented yet, right ? Just dummy properties so far ? 
sorry if missing something.

Reply:
@narek-h, yes, you are right.

Reply:
@narek-h, thanks! Added record to CMakeLists.txt